src/Manifest.js

```diff
@@ -283,6 +292,13 @@ function Manifest(output, path) {
 Manifest.ANDROID_DEFAULT_PERMISSIONS = [ "ACCESS_NETWORK_STATE", "ACCESS_WIFI_STATE", "INTERNET" ];
 
 /**
+ * Default codepage to use is "western european".
+ * When using ISO 8859-1 unsupported characters in the manifest,
+ * the codepage needs to be adjusted accordingly.
+ */
+Manifest.WINDOWS_DEFAULT_CODEPAGE = "1252";
```

Comment:
Might have to merge those two tables and use it then.
https://msdn.microsoft.com/en-us/library/ee825488(v=cs.20).aspx
https://msdn.microsoft.com/en-us/library/ms930130.aspx